Emrald-UI/src/components/diagrams/EmraldDiagram/useContextMenu.tsx

```diff
@@ -550,7 +537,7 @@ const useContextMenu = (
         (state) => state !== itemToDelete.name,
       );
       getStateNodes();
-      updateDiagramDetails(currentDiagram.value);
+      // updateDiagramDetails(currentDiagram.value);
```

Comment:
I think we can just remove this. If updateDiagramDetails is still in the context we need to remove it there also.